nilearn/decoding/tests/test_decoder.py

```diff
@@ -187,6 +188,66 @@ def test_check_parameter_grid_is_empty(rand_X_Y):
     assert param_grid == {}
 
 
+@pytest.mark.parametrize(
+    "param_grid",
+    [
+        {"alphas": [1, 10, 100, 1000]},
+        {"alphas": [1, 10, 100, 1000], "fit_intercept": [True, False]},
+        {"fit_intercept": [True, False]},
+        {"alphas": [[1, 10, 100, 1000]]},
+        {"alphas": (1, 10, 100, 1000)},
+        {"alphas": [(1, 10, 100, 1000)]},
+        {"alphas": ((1, 10, 100, 1000),)},
+        {"alphas": np.array([1, 10, 100, 1000])},
+        {"alphas": [np.array([1, 10, 100, 1000])]},
+        [{"alphas": [1, 10]}, {"alphas": [[100, 1000]]}],
+        [{"alphas": [1, 10]}, {"fit_intercept": [True, False]}],
+    ],
+)
+def test_wrap_param_grid(param_grid):
+    param_name = "alphas"
+    for param in ParameterGrid(_wrap_param_grid(param_grid, param_name)):
+        try:
+            for param_value in param[param_name]:
```

Comment:
here the goal is to check that `param["alpha"]` is a list of numbers?

Reply:
Yes, but another way (if we want to avoid hardcoding the values) would be to use `isinstance(param["alpha"], (float, int))`. Do you think that would be better?